utils/model_utils.py

```
@@ -339,18 +344,82 @@ def get_existing_models() -> Generator[Model, None, None]:
+        for dataset in datasets:
+            print(f"  {dataset.info.dataset_name}")
+            output_directory = (
+                BASE_DIRECTORY
+                / "models_custom"
+                / output_path
+                / model_name.split("/")[-1]
+                / dataset.info.dataset_name
+            )
+
+            tokenized_dataset = dataset.map(
+                lambda batch: tokenizer(
```

Ruff (0.8.0)

Function definition does not bind loop variable `tokenizer`

(B023)